samcli/lib/build/app_builder.py

```diff
@@ -676,23 +704,32 @@ def _build_function(  # pylint: disable=R1710
     def _get_build_options(
         function_name: str,
         language: str,
+        base_dir: str,
         handler: Optional[str],
         dependency_manager: Optional[str] = None,
         metadata: Optional[dict] = None,
+        source_code_path: Optional[str] = None,
+        scratch_dir: Optional[str] = None,
     ) -> Optional[Dict]:
         """
         Parameters
         ----------
         function_name str
-            currrent function resource name
+            current function resource name
         language str
             language of the runtime
+        base_dir : str
```

Comment:
nit: consistency with other line in this docstring
```suggestion
        base_dir str
```

Reply:
or vice versa - others should change to "name : type"

Reply:
will fix it

Reply:
resolved